earth_enterprise/src/fusion/autoingest/plugins.src/MercatorRasterProject.src

```diff
@@ -868,9 +865,10 @@ uint32 MercatorRasterProjectAssetVersionImplD::BuildIndex(
   if (type == AssetDefs::Imagery) {
     // first we know that we don't want skip tiles during minification
     // so set each inset to start after that
-    for (uint i = 0; i < genInfo.size(); ++i) {
-      genInfo[i].beginSkipTransparentLevel =
-        std::max(endMinifyLevel, genInfo[i].coverage.beginLevel());
+
+    for (auto& i : genInfo) {
+        i.beginSkipTransparentLevel =
```

Comment:
Minor: would be good to reduce this indentation by 2 spaces so that the next line is indented more than this one.